### PR TITLE
Update framework-components.md in tandem with #371

### PR DESCRIPTION
**Pull request by leader22**
Refs  #364 + #371

### CODE REVIEW

**Comment by Jutanium** [comment]
This change needs to happen for sure. I don't think it will let me edit the PR but this is how I'd tweak:

Each framework has its own patterns for nesting: `children` and render props for React and Solid; slots for Svelte and Vue, for example. Note, however, that when you include a framework component in an astro component, you can't use render props or named slots, even if the framework component supports it. This is due to a limitation in Astro's compiler.

**Comment by leader22** [comment]
@Jutanium Thank you for your suggestion! I updated. 👍🏻

**Comment by Jutanium** [comment]
I don't have access on this PR, but we can remove unnecessary italics:


Each framework has its own patterns for nesting: `children` props and render props for React and Solid; `<slot>` with or without names for Svelte and Vue, for example. Note, however, that when you include a framework component in an astro component, you can't use render props or named slots, even if the framework component supports it. This is due to a limitation in Astro's compiler.

**Comment by sarah11918** [comment]
@Jutanium Do you have access to go into Files Changed and click to make a suggestion there? That makes it easy for me to "commit suggestion" and merge. (I'll ping you on Discord to make sure you're able to do this, since it will help going forward!)

File: src/pages/en/core-concepts/framework-components.md [hunk 1]
```diff
@@ -121,9 +120,11 @@ import MyButton from '../components/MyButton.svelte';
...
+⚠️ *Note that framework components themselves (e.g. `.jsx`, `.svelte`) cannot mix multiple frameworks.*
+
+This allows you to build entire "apps" in your preferred JavaScript framework and render them, via a parent component, to an Astro page. This is a convenient pattern to allow related components to share state or context.
 
-This allows you to build entire "apps" in your preferred JavaScript framework and render them, via a parent component, to an Astro page. This is a convenient pattern to allow related components to share state or context. 
+Each framework has its own patterns for nesting: `children` props and _render props_ for React and Solid; `<slot>` with or without names for Svelte and Vue, for example. Note, however, that when you include a framework component in an astro component, you can't use _render props_ or _named slots_, even if the framework component supports it. This is due to a limitation in Astro's compiler.
```
Comment (Jutanium) [inline src/pages/en/core-concepts/framework-components.md:127]: Each framework has its own patterns for nesting: `children props` and render props for React and Solid; `<slot>` with or without names for Svelte and Vue, for example. Note, however, that when you include a framework component in an astro component, you can't use render props or named slots, even if the framework component supports it. This is due to a limitation in Astro's compiler.

Comment (sarah11918) [inline src/pages/en/core-concepts/framework-components.md:127]:
```suggestion
Each framework has its own patterns for nesting: `children` props and _render props_ for React and Solid; `<slot>` with or without names for Svelte and Vue, for example. Note, however, that when you include a framework component in an astro component, you can't use render props or named slots, even if the framework component supports it. This is due to a limitation in Astro's compiler.
```

File: src/pages/en/core-concepts/framework-components.md [hunk 2]
```diff
@@ -121,9 +120,11 @@ import MyButton from '../components/MyButton.svelte';
...
+⚠️ *Note that framework components themselves (e.g. `.jsx`, `.svelte`) cannot mix multiple frameworks.*
+
+This allows you to build entire "apps" in your preferred JavaScript framework and render them, via a parent component, to an Astro page. This is a convenient pattern to allow related components to share state or context.
 
-This allows you to build entire "apps" in your preferred JavaScript framework and render them, via a parent component, to an Astro page. This is a convenient pattern to allow related components to share state or context. 
+Each framework has its own patterns for nesting: `children` props and _render props_ for React and Solid; `<slot>` with or without names for Svelte and Vue, for example. Note, however, that when you include a framework component in an astro component, you can't use render props or named slots, even if the framework component supports it. This is due to a limitation in Astro's compiler.
```
Comment (sarah11918) [inline src/pages/en/core-concepts/framework-components.md:127]:
```suggestion
Each framework has its own patterns for nesting: `children` props and render props for React and Solid; `<slot>` with or without names for Svelte and Vue, for example. Note, however, that when you include a framework component in an astro component, you can't use render props or named slots, even if the framework component supports it. This is due to a limitation in Astro's compiler.
```

File: src/pages/en/core-concepts/framework-components.md [hunk 3]
```diff
@@ -124,7 +124,7 @@ import MyButton from '../components/MyButton.svelte';
 
 This allows you to build entire "apps" in your preferred JavaScript framework and render them, via a parent component, to an Astro page. This is a convenient pattern to allow related components to share state or context.
 
-Each framework has its own patterns for nesting: `children` props and _render props_ for React and Solid; `<slot>` with or without names for Svelte and Vue, for example. Note, however, that when you include a framework component in an astro component, you can't use render props or named slots, even if the framework component supports it. This is due to a limitation in Astro's compiler.
+Each framework has its own patterns for nesting: `children` props and render props for React and Solid; `<slot>` with or without names for Svelte and Vue, for example. Note, however, that when you include a framework component in an astro component, you can't use render props or named slots, even if the framework component supports it. This is due to a limitation in Astro's compiler.
```
Comment (Jutanium) [inline src/pages/en/core-concepts/framework-components.md:127]:
```suggestion
Each framework has its own patterns for nesting: `children` props and [render props](https://reactjs.org/docs/render-props.html) for React and Solid; `<slot>` with or without names for Svelte and Vue, for example. Note, however, that you can't pass render props or named slots to a framework component from a `.astro` file, even if the framework component supports it. This is due to a limitation in Astro's compiler.
```

**Comment by sarah11918** [comment]
Merging this evil PR.

**Comment by sarah11918** [comment]
(I mean, THANKS EVERYONE!! ❤️ )

**Pull request merged**